modules/javafx.graphics/src/main/java/javafx/application/Platform.java

```diff
@@ -432,4 +436,280 @@ public static ReadOnlyBooleanProperty accessibilityActiveProperty() {
         }
         return accessibilityActiveProperty.getReadOnlyProperty();
     }
+
+    /**
+     * Gets the preferences of the current platform.
+     * <p>
+     * The map returned from this method is unmodifiable, which means that keys and values cannot
+     * be added, removed, or updated. Calling any mutator method on the map will always cause
+     * {@code UnsupportedOperationException} to be thrown. However, the mappings will be updated
+     * by JavaFX when the operating system reports that a platform preference has changed.
+     *
+     * @return the {@code Preferences} instance
+     * @see <a href="Platform.Preferences.html#preferences-table">List of platform preferences</a>
+     * @since 22
+     */
+    public static Preferences getPreferences() {
+        return PlatformImpl.getPlatformPreferences();
+    }
+
+    /**
+     * Contains UI preferences of the current platform.
+     * <p>
+     * {@code Preferences} extends {@link ObservableMap} to expose platform preferences as key-value pairs.
+     * For convenience, {@link #getInteger}, {@link #getDouble}, {@link #getBoolean}, {@link #getString},
+     * {@link #getColor}, {@link #getPaint}, and {@link #getValue} are provided as typed alternatives to
+     * the untyped {@link #get} method.
+     * <p>
+     * The preferences that are reported by the platform may be dependent on the operating system version,
+     * so applications should not assume that a particular preference is always available.
+     * <p>
+     * The following list contains all preferences that are potentially available on the specified platforms:
+     * <table id="preferences-table">
```

Comment:
observation: this table does not render correctly in eclipse -

<img width="284" alt="Screenshot 2023-09-06 at 10 26 37" src="https://github.com/openjdk/jfx/assets/107069028/47d0d778-366b-438a-8162-848b80a202be">

but it renders ok in the browser.  so might be an eclipse issue, unless we are missing something.